src/server.rs

```diff
@@ -2604,9 +2603,6 @@ errorCounts
             items.iter().map(|i| i.label.as_str()).collect();
 
         let want: HashSet<&str> = vec![
-            "_fillEmpty",
-            "_highestOrLowest",
-            "_sortLimit",
```

Comment:
Would it make sense to move these tests to the `completion` package?

Reply:
I _think_ that's right, though these currently serve more as "integration tests" in that they assert on the higher level, exported behavior. The completion package has a lot of technical debt that makes it ripe for refactor, but these tests mean we can ensure we didn't break anything as far as the client interaction.